torchrec/distributed/utils.py

```diff
@@ -34,7 +34,7 @@
         "//deeplearning/fbgemm/fbgemm_gpu:permute_pooled_embedding_ops_cpu"
     )
     torch.ops.load_library(
-        "//deeplearning/fbgemm/fbgemm_gpu:permute_pooled_embedding_ops_gpu"
+        "//deeplearning/fbgemm/fbgemm_gpu:permute_pooled_embedding_ops_cuda"
```

Comment:
I feel we may consider just have "//deeplearning/fbgemm/fbgemm_gpu:permute_pooled_embedding_ops"